thingsboard_gateway/config/ble.json

```diff
@@ -13,6 +13,7 @@
       "pollPeriod": 500000,
       "showMap": false,
       "timeout": 10000,
+      "connectRetry": 5,
       "telemetry": [
```

Comment:
Please add something like "wipeConnectionTriesInSeconds" and set connect try to 0 after this time. It will give the ability to reconnect to device after some period.